lib/src/core/transport.dart

```diff
@@ -126,10 +127,18 @@ class Transport extends Disposable {
       }
     }
 
+    if (restartingIce && !rtc.WebRTC.platformIsWeb) {
```

Comment:
how do we handle this for web?

Reply:
On the web, it will be triggered by `pc.createOffer({iceRestart: true});`, only in flutter native need to call `pc.restartIce();`